deps/blas.mk

```diff
@@ -46,7 +46,7 @@ $(BUILDDIR)/$(OPENBLAS_SRC_DIR)/build-compiled: | $(BUILDDIR)/objconv/build-comp
 endif
 endif
 
-OPENBLAS_FFLAGS :=
+OPENBLAS_FFLAGS := $(FFLAGS) $(JFFLAGS)
 OPENBLAS_CFLAGS :=
```

Comment:
Hm. Why wouldn't the opt flag have been set at https://github.com/xianyi/OpenBLAS/blob/v0.2.19/Makefile.system#L992-L1005 then?